src/ui/field_formats_mixin.js

```diff
@@ -22,7 +22,7 @@ export function fieldFormatsMixin(kbnServer, server) {
     return new FieldFormatsService(fieldFormatClasses, getConfig);
   });
 
-  server.decorate('server', 'registerFieldFormatClass', (FieldFormat) => {
+  server.decorate('server', 'registerFieldFormat', (FieldFormat) => {
```

Comment:
Field formatters were moved to the server so they could be used for generating CSV reports.

Reply:
> If there's no reason to do that, then I'd say we can just consider the whole feature broken and changing the name here is fine as part of the fix.

That's how I see it. People could add formats that are browser only, but when they do that they won't be usable in CSV export. That should be _fine_, because the default format for the field type should be used, but since they weren't able to add field formats for the UI anyway I think the feature was broken enough that this isn't a BWC concern